launcher/plans/translator_nest_to_tvb_on_local.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<settings>
+    <co_simulation_action_plan>
+        <title>Test the translator Nest to TVB</title>
+        <description>Test the translator Nest to TVB</description>
+        <parameters>
```

Comment:
Did you forget to name it transformer? Also the file name itself contains 'translator'

Reply:
I TOTALLY agree on it. 

Addtionally, it will worth to agree on the proper naming convention for xml files and some python scripts where the science is shaped.